src/locales/en.json

```diff
@@ -1539,6 +1549,7 @@
 "label.region": "Region",
 "label.region.details": "Region details",
 "label.reinstall.vm": "Reinstall VM",
+"label.reject": "Reject",
```

Comment:
check old UI, for add network offering; if reject is used or something else?

Reply:
Was reject itself, but as an inline string